src/components/PathwaysCard/PathwaysCard.jsx

```diff
@@ -0,0 +1,84 @@
+import { React, useState } from "react";
+
+import { CircularProgressbar, buildStyles } from "react-circular-progressbar";
+import "react-circular-progressbar/dist/styles.css";
+import "./PathwaysCard.scss";
+import image from "../../assets/images/financial-forest.svg";
```

Comment:
This should be a prop that is passed into the component.
So when we create multiple cards we give it the data it needs, rather than hardcoding it.